tartiflette/parser/visitor.py

```diff
@@ -107,32 +110,31 @@ def _on_variable_in(self, element):
         except KeyError:
             raise UnknownVariableException(var_name)
 
-    def _find_resolver(self, path_name):
-        try:
-            return self._schema_definition.get_resolver(path_name)
-        except KeyError:
-            pass
+    def _find_resolver(self, element_name: str):
+        parent_name = ""
+        if self._parent_node_type is None:
+            parent_name = self._schema_definition.query_type
+        else:
+            parent_name = self._parent_node_type
 
-        try:
-            return self._schema_definition.get_resolver(
-                "%s/%s" % (self._current_node.name, path_name),
-            )
-        except (KeyError, AttributeError):
-            # AttributeError for no Parent
-            # KeyError no resolver
-            pass
+        field_name = parent_name + '.' + element_name
+        field = self._schema_definition.get_field_by_name(field_name)
 
-        return _default_resolver
+        if field:
+            self._parent_node_type = reduce_type(field.gql_type)
+            return field.resolver
+        else:
```

Comment:
can you try: except: this instead of lybl ?

Reply:
Fixed in last commit :)